Source/WebKit/Shared/RemoteLayerTree/DynamicContentScalingImageBufferBackend.mm

```diff
@@ -128,6 +128,10 @@ static CFDictionaryRef makeContextOptions(const DynamicContentScalingImageBuffer
                 String objectDescription { adoptCF(CFCopyDescription(port)).get() };
                 auto description = makeString("Unexpected type in DCS ports array: "_s, typeID, " "_s, typeDescription, " "_s, objectDescription);
                 logAndSetCrashLogMessage(description.utf8().data());
+
+                std::array<uint64_t, 6> values { 0, 0, 0, 0, 0, 0 };
+                strncpy(reinterpret_cast<char*>(values.data()), typeDescription.utf8().data(), sizeof(values));
+                CRASH_WITH_INFO(values[0], values[1], values[2], values[3], values[4], values[5]);
```

Comment:
Maybe someone should add `crashWithClassName()` to `CocoaHelpers` or `Assertions`

```cpp
NO_RETURN void crashWithClassName(Class class)
{
    std::array<uint64_t, 6> values { 0, 0, 0, 0, 0, 0 };
    strncpy(reinterpret_cast<char*>(values.data()), NSStringFromClass(objectClass).UTF8String, sizeof(values));
    CRASH_WITH_INFO(values[0], values[1], values[2], values[3], values[4], values[5]);
}
```

Reply:
yes, it may be time to hoist it